### ISSUE TITLE
Pick and then fix/change what happens to user's name on the Recent Listings pane

### ISSUE THREAD

**Issue by JSlote** [CarletonDevX/textbook-exchange#163]

**Comment by taoduo** [comment]
Do you mean the names of the sellers? What do you want to reset the names to? Maybe the names can not be seen after log out?

**Comment by JSlote** [comment]
Right now I'm using the fullname attribute on the recent listings view. When a user isn't logged in, the server only outputs the user's given name as `fullName` to preserve privacy, but when a user is logged in `fullName` is `givenName + ' ' + familyName`. Right now if you log in, the names stay as just first names until you refresh the page. So that's a bug (it should resend the request to get everything directly on login).

On the other hand, David thought we should just use first names always.

**Comment by JSlote** [comment]
I agree. However, I think because the transactions occur face-to-face, it's really important to communicate specificity and authenticity the users, especially if they're new.
